.github/dependabot.yml

```diff
@@ -291,6 +225,17 @@ updates:
       include: "scope"
     reviewers:
       - "ministryofjustice/data-platform-core-infra"
+  - package-ecosystem: "terraform"
+    directory: "terraform/aws/analytical-platform-development/cluster/.terraform/modules/iam_assumable_role_ebs_csi_driver/_template/test"
+    schedule:
+      interval: "daily"
+      time: "09:00"
+      timezone: "Europe/London"
+    commit-message:
+      prefix: "terraform"
+      include: "scope"
+    reviewers:
+      - "ministryofjustice/data-platform-core-infra"
```

Comment:
this has been picked up because there is a `.terraform.lock.hcl` in `terraform/aws/analytical-platform-development/cluster/.terraform/modules/iam_assumable_role_ebs_csi_driver/_template/test`, you'll need to delete `terraform/aws/analytical-platform-development/cluster/.terraform` and run again